.gitlab-ci.yml

```diff
@@ -174,29 +154,12 @@ pup5.5.7-unit:
   <<: *pup_5_5_7
```

Comment:
Please test with Puppet 5.5.10 and Ruby 2.4.5

Reply:
Updated and pushed to https://gitlab.com/simp/pupmod-simp-deferred_resources/pipelines/55208552 but the likelihood of breaking changes is low and we should probably wait on these for a full static assets bump instead of blocking work in general. Things will be missed and that's OK until we can blanket push in general.

Reply:
:heavy_check_mark: https://gitlab.com/simp/pupmod-simp-deferred_resources/pipelines/55208552